3_tests/immo/input/assert_one_price_mutation.sql

```diff
@@ -10,8 +10,12 @@ WITH mutation_values AS (
         latitude,
         longitude,
         valeur_fonciere
-    FROM 
+    FROM
+    {% if target.name == 'production' %}
```

Comment:
Probably we need to add the right target to all the `dbt test` calls?
What about running `dbt build` instead of seed+run+test instead?